Dockerfile

```diff
@@ -17,8 +17,10 @@ RUN CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-w -s' -a -installsuffix cgo
 
 
 FROM alpine:3.7
-ENV XDG_CONFIG_HOME=/che-jwtproxy-config/
-VOLUME /che-jwtproxy-config
 COPY --from=builder /go/src/github.com/eclipse/che-jwtproxy/jwtproxy /usr/local/bin
 ENTRYPOINT ["jwtproxy"]
-CMD ["-config", "/che-jwtproxy-config/config.yaml"]
+# The JWT proxy needs 2 things:
+# * the location of the configuration file supplied as an argument:
+#   `-config <location/of/the/config.yaml>`
+# * The XDG_CONFIG_HOME environment variable pointing to a directory where to store auth keys
+# CMD ["-config", "/che-jwtproxy-config/config.yaml"]
```

Comment:
Would it be worth explaining HOW these values will be set in future, if not hardcoded here? 

I'm worried that in future when we remove this, knowing how/where to debug/look at upstream code that calls jwtproxy might be challenging.

Reply:
How these things are applied depends on the user of the image. In case of Che, it is the `JwtProxyProvisioner` (and I can add a note here to point to it), but I believe https://github.com/che-incubator/che-workspace-operator will also use this image in the future.